.github/workflows/build.yml

```diff
@@ -0,0 +1,97 @@
+name: alfresco Extension Inspector Build Workflow
+
+on:
+  push:
+    branches:
+      - master
+      - 'fix/**'
+      - 'support/**'
+      - 'feature/**'
+  pull_request:
+    types: [opened, synchronize]
```

Comment:
why overriding the default that also includes reopen?

Reply:
the github action for pull request has the activity types mentioned
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request
So will the build trigger for all activity types by default.

Reply:
not all but the most common/relevant:
> By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened.

i.e. you never need to specify types unless you are addressing very specific use cases